extensions/random/package.json

```
@@ -0,0 +1,48 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "random",
+  "title": "Random",
```

```suggestion
  "title": "Dice & Coin",
```
Could be a better name?